Gemfile

```diff
@@ -54,7 +62,10 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'pry', '~> 0.10.4'
+  gem 'better_errors'
 
+  gem 'rb-readline', platforms: :ruby, install_if: os_is(/darwin/)
```

Comment:
I'm not understanding maybe? But its your ruby installation that's broken and requiring this config correct? If so why you need this in the app? If you use a higher ruby version is this needed? If you use rvm instead of chruby maybe? Looks like you can just `brew install readline` instead from quickly googling this https://github.com/guard/guard/wiki/Add-Readline-support-to-Ruby-on-Mac-OS-X?

 I mean if you need it to run the app then fine add it. But please document this? Cause there is nothing about what this is for or what this does. Other then it is required for Mac OSX users which isn't exactly true? My Macbook at home can run ruby/rails code without this in the gemfile? And I have never seen popular open source rails apps requiring this either in their gemfiles (and most people dev in Mac/OSX)